### DISCUSSION TITLE
Make Travel Form Scale With Riding Skill Like Flight Form

### DISCUSSION THREAD

**Discussion by NinjaWitaSniper** [chromiecraft/chromiecraft#4846] [Suggestions]
How possible is this?

**Comment by 55Honey** [comment]
Hi @NinjaWitaSniper ,
This is a very not-blizzlike suggestion. The first question is why we should do it when it's not at all meant to be like that?

**Reply by NinjaWitaSniper** [reply]
Sorry if my original reply came off as hostile, that was not my intention.

I wasn't aware that being strict blizzlike would be a barrier for a change like this. Theres vendors selling glyphs we cant make, custom boss events, teleporting pvp events, and portals everywhere, those changes arent strictly blizzlike either?

**Reply by 55Honey** [reply]
Thanks for the follow-up.
The custom events we have introduced do not affect the blizzlike progression. And the glyph vendor actually serves to make blizzlike progression possible by un-gimping certain specs. None of that affects balancing much.

Travel form is a spell used in PvP. It would massively impact the balancing of PvP.

**Reply by NinjaWitaSniper** [reply]
I didn't think about the pvp side of things when I thought about that, thats actually terrible. Thanks for the response.
